addons/movments_accounts_balances/controllers/controllers.py

```diff
@@ -97,108 +98,663 @@ def general_ledger_report(self, account_id, start_date, end_date):
             'ledger_data': ledger_data,
         }
 
+    ##create/get/delete_bills
+    @api.model
+    def create_bill(self, invoice_vals, partner_id, invoice_date, invoice_date_due, reference, narration):
+        """
+        Create an AR invoice and corresponding analytic lines based on the provided data.
+        """
+        # Prepare the invoice lines
+        invoice_lines = []
+        for line in invoice_vals:
+            invoice_line_vals = {
+                'name': line.get('description', ''),
+                'quantity': line.get('quantity', 1.0),
+                'price_unit': line.get('price_unit', 0.0),
+                'account_id': line.get('account_id'),
+                'product_id': line.get('product_id', False),
```

Comment:
We want to avoid product. Can we avoid this. The create_bill method in my PR didn't have any product: https://github.com/sugampandey/odoo/pull/12/files